scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,157 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    group_role TEXT;
+BEGIN
+    group_role := cartodb._CDB_Group_GroupRole(group_name);
+    PERFORM cartodb._CDB_Group_CreateGroup_API(current_database(), group_name, group_role);
```

Comment:
Seeing that function call I'm not sure why it's required. Should its name look like a *notify* thingy?